run.py

```diff
@@ -77,6 +77,7 @@
         [--docker-image <image>]
         [--docker-tag <tag>]
         [--insecure-registry]
+        [--install-license]
```

Comment:
Instead of new arg, can we use `--custom-config ibm-build=True`?. However we have to go through new args for IBM later for complete pipeline.

Reply:
Let's not add anything in the run.py. so it would be easy to decouple this when we have proper  solution.

Reply:
@sunilkumarn417 thanks for feedback. Updated code accordingly.